docs/user-guide/tokens.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: main
+title: Access Tokens
+category: User Guide
+menu: menu
+toc:
+    - title: Access Tokens
+      url: "#access-tokens"
+    - title: User Access Token
+      url: "#user-access-token"
+    - title: Pipeline Access Token
+      url: "#pipeline-access-token"
+    - title: Build Access Token
+      url: "#build-access-token"
+---
+# Access Tokens
+There are different access tokens users can use to make calls to the Screwdriver API varying levels of scope: build, pipeline, and user.
+
+To authenticate with your newly created token, make a GET request to `https://${API_URL}/v4/auth/token?api_token=${YOUR_TOKEN_VALUE}`. This returns a JSON object with a token field. The value of this field will be a JSON Web Token, which you can use in an Authorization header to make further requests to the Screwdriver API. This JWT will be valid for 12 hours, after which you must re-authenticate. See the [API documentation](./api) for more details.
```

Comment:
this section is applicable only for User & Pipeline Tokens. Build token is already an API JWT token.